compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -135,7 +98,9 @@ def AMDAIEFusePackIntoLoop :
     Option<"fusePackDepth", "fuse-pack-depth", "int64_t", /*default=*/"1",
       "Set the depth until which we would keep fusing producer tensor.pack chain">,
     Option<"useSCFFor", "use-scf-for", "bool", /*default=*/"true",
-      "Set the innermost scf loop type to fuse tensor.pack ops into">
+      "Set the innermost scf loop type to fuse tensor.pack ops into">,
+    Option<"targetElementwise", "target-elementwise", "bool", /*default=*/"false",
+      "Set if the compute op within the loop is an elementwise op">
```

Comment:
`Set if the compute op within the loop is an elementwise op` -> `Set if the target compute op within the loop is an elementwise op`.